Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Image to build from
+FROM node:10
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Copy package.json
+COPY package*.json ./
+
+# Install all dependencies
+RUN npm install
+
+# Bundle app source
+COPY . .
+
+# Map port to the Docker daemon
+EXPOSE 3000
+
+# Run the start script
+CMD [ "npm", "dev" ]
```

Comment:
you are running  ```shell npm dev ``` inside docker container but right now we don't have it in package.json either put it there or this PR has to wait for @amilykassim's work to get marge. thanks

Reply:
That was the idea, waiting for @amilykassim PR, but let me just add it there too, whatever PR that get merged first we'll know what to do